chisel/src/driver.rs

```diff
@@ -184,6 +205,278 @@ impl ChiselDriver {
         self.state = DriverState::Done(results);
         &self.state
     }
+
+    pub fn execute_module(
+        &mut self,
+        name: String,
+        module: ModuleConfig,
+        wasm: &mut Module,
+    ) -> Result<ModuleResult, DriverError> {
+        let result = match name.as_str() {
+            "checkfloat" => {
+                let checkfloat = CheckFloat::new();
+                let module_result = checkfloat.validate(wasm);
+                ModuleResult::Validator(name.clone(), module_result)
+            }
+            "checkstartfunc" => {
+                if let Some(require_start) = module.options().get("require_start") {
+                    let require_start = match require_start.as_str() {
+                        "true" => true,
+                        "false" => false,
+                        _ => {
+                            return Err(DriverError::InvalidField(
+                                name.clone(),
+                                "require_start".to_string(),
+                            ));
+                        }
+                    };
+                    let checkstartfunc = CheckStartFunc::new(require_start);
+                    let module_result = checkstartfunc.validate(wasm);
+                    ModuleResult::Validator(name.clone(), module_result)
+                } else {
+                    chisel_debug!(1, "checkstartfunc missing field 'require_start'");
+                    return Err(DriverError::MissingRequiredField(
+                        name.clone(),
+                        "require_start".to_string(),
+                    ));
+                }
+            }
+            "deployer" => unimplemented!("Creator modules are not supported yet."),
+            "dropsection" => {
```

Comment:
Suggest this implementation is dropped as-is given its not there in the current CLI.